databricks/koalas/frame.py

```diff
@@ -4229,6 +4229,62 @@ def head(self, n=5):
 
         return DataFrame(self._internal.copy(sdf=self._sdf.limit(n)))
 
+    def tail(self, n=5):
+        """
+        Return the last `n` rows.
+
+        This function returns the last `n` rows for the object based
+        on position. It is useful for quickly verifying data,
+        for example, after sorting or appending rows.
+
```

Comment:
FYI, I plan to propose a new API `tail` in Spark itself. If we manage to add it, we could efficiently do that.

Reply:
Can we hold on for a while? We could leave a note about that if we add it in Spark.

Reply:
@HyukjinKwon ah, that sounds great!! 👍 
Okay, then just let me know for fix this after `tail` will be added on spark,

Reply:
PR submitted at https://github.com/apache/spark/pull/26809. Let's see how it goes.

Reply:
@HyukjinKwon awesome! thanks for reminding me 👍